base/uk.ac.stfc.isis.ibex.opis/resources/PEARLPC.opi

```diff
@@ -3369,24 +3369,24 @@ $(pv_value)</tooltip>
           </font>
           <forecolor_alarm_sensitive>false</forecolor_alarm_sensitive>
           <foreground_color>
-            <color name="ISIS_Standard_Text" red="0" green="0" blue="0"/>
+            <color name="ISIS_Standard_Text" red="0" green="0" blue="0" />
           </foreground_color>
           <format_type>0</format_type>
           <height>34</height>
           <horizontal_alignment>0</horizontal_alignment>
           <name>Interface_Status_Text</name>
           <precision>0</precision>
           <precision_from_pv>true</precision_from_pv>
-          <pv_name>$(PV_ROOT_PEARLPC):record_name</pv_name>
-          <pv_value/>
+          <pv_name>$(PV_ROOT_PEARLPC):STATUS_ARRAY</pv_name>
```

Comment:
This is just going to display an array of integers. Doesn't seem very readable, is this really what we want here?

Reply:
I agree about the readability, but I believe that's what's supposed to be returned from looking at the LabVIEW VI panel mentioned [HERE](https://github.com/ISISComputingGroup/IBEX/issues/6040#issuecomment-889145254).